Edits to a couple format and handling pages as part of site review (#196)
pinot-contrib/pinot-docs: basics/data-import/pinot-input-formats.md

  multiValueDelimiter: '-'
```

CSV Record Reader supports the following configs:

> * **`fileFormat`**: Can be one of: default, rfc4180, excel, tdf, mysql.

Reviewer: ```suggestion
* **`fileFormat`**: `default`, `rfc4180`, `excel`, `tdf`, `mysql`
```

> * **`header`**: Header of the file. The columnNames should be seperated by the delimiter mentioned in the configuration.

Reviewer: ```suggestion
* **`header`**: Header of the file. The `columnNames` should be separated by the delimiter mentioned in the configuration.
```

* **`delimiter`**: The character seperating the columns.

> * **`multiValueDelimiter`**: The character seperating multiple values in a single column. This can be used to split a column into a list.

Reviewer: ```suggestion
* **`multiValueDelimiter`**: The character separating multiple values in a single column. This can be used to split a column into a list.
```

Reviewer: lets run spell check as part of the first pass to be sure

> Supported from 0.11 release:

Reviewer: ```suggestion
```

> * **`skipHeader`**: skip header record in the file. Boolean

Reviewer: ```suggestion
* **`skipHeader`**: Skip header record in the file. Boolean.
```

> * **`ignoreEmptyLines`**: ignore empty lines instead of consuming them and filling with default values. Boolean

Reviewer: ```suggestion
* **`ignoreEmptyLines`**: Ignore empty lines (instead of filling them with default values). Boolean.
```

* **`ignoreSurroundingSpaces`**: ignore spaces around column names and values. Boolean

> * **`quoteCharacter`**: single character that is being used for quotes in CSV files

Reviewer: ```suggestion
* **`quoteCharacter`**: Single character used for quotes in CSV files.
```

> * **`recordSeparator`**: character used to seperate records in the input file. Default is `\n` or `\r\n` depending on the platform.

Reviewer: ```suggestion
* **`recordSeparator`**: Character used to separate records in the input file. Default is `\n` or `\r\n` depending on the platform.
```

    enableLogicalTypes: true
```

> The Avro record reader converts the data in file to a `GenericRecord`. A Java class or `.avro` file is not required. By default the avro record reader only supports primitive types. You can set `enableLogicalTypes` to `true` to enable support for rest of the Avro data types.

Reviewer: ```suggestion
The Avro record reader converts the data in file to a `GenericRecord`. A Java class or `.avro` file is not required. By default, the Avro record reader only supports primitive types. To enable support for rest of the Avro data types, set `enableLogicalTypes` to `true` .
```

> We use the following conversion table to translate between Avro and Pinot data types. The conversions are done using the offical Avro methods present in `org.apache.avro.Conversions`

Reviewer: ```suggestion
We use the following conversion table to translate between Avro and Pinot data types. The conversions are done using the offical Avro methods present in `org.apache.avro.Conversions`.
```

```

{% hint style="info" %}

> Thrift requires the generated class using `.thrift` file to parse the data. The .class file should be available in the Pinot's classpath. You can put the files in the `lib/` folder of Pinot distribution directory.

Reviewer: ```suggestion
Thrift requires the generated class using `.thrift` file to parse the data. The `.class` file should be available in the Pinot's `classpath`. You can put the files in the `lib/` folder of Pinot distribution directory.
```

className: 'org.apache.pinot.plugin.inputformat.parquet.ParquetRecordReader'
```

> Since 0.11.0 release, The Parquet record reader determines whether to use `ParquetAvroRecordReader` or `ParquetNativeRecordReader`  to read records. The reader looks for `parquet.avro.schema` or `avro.schema` key in the parquet file footer and if present uses the Avro reader.

Reviewer: ```suggestion
Since 0.11.0 release, the Parquet record reader determines whether to use `ParquetAvroRecordReader` or `ParquetNativeRecordReader` to read records. The reader looks for the `parquet.avro.schema` or `avro.schema` key in the parquet file footer, and if present, uses the Avro reader.
```

> Users can however change the record reader manually in case of a misconfiguration.

Reviewer: ```suggestion
You can change the record reader manually in case of a misconfiguration.
```

```
dataFormat: 'parquet'
className: 'org.apache.pinot.plugin.inputformat.parquet.ParquetNativeRecordReader'
```

{% hint style="warning" %}

> For the support of DECIMAL and other parquet native data types, always use `ParquetNativeRecordReader`

Reviewer: ```suggestion
For the support of DECIMAL and other parquet native data types, always use `ParquetNativeRecordReader`.
```
